frontend/src/store/screenSizeState.ts

```diff
@@ -0,0 +1,20 @@
+import { sideNavState } from "@/store/SideNavState";
+import { setNotificationMsg, newMsgList, newFormData, newSeverity } from "./NotificationState";
+
+export let screenSize = window.innerWidth;
+
+window.addEventListener("resize", (event) => {
+    screenSize = window.innerWidth;
+
+    //update the message when the screensize change
+    if(newMsgList.value) {
+        setNotificationMsg(newMsgList.value, newFormData.value, newSeverity.value);
```

Comment:
and here, you can create a for loop:

Object.keys(notifications.value).forEach((severity) => {
    if(notifications.value[severity].msg &&  notifications.value[severity].forestClientNumberList > 1) {
        setNotificationMsg(notifications.value[severity].forestClientNumberList, severity);
    }
})